kickstart/provision.erb

```diff
@@ -143,6 +144,11 @@ echo "updating system time"
 # update all the base packages from the updates repository
 yum -t -y -e 0 update
 
+<% if chef_enabled %>
+echo "Bootstraping chef"
+<%= respond_to?(:chef_bootstrap) ? chef_bootstrap(@host) : snippet_if_exists("chef-client omnibus bootstrap") %>
```

Comment:
They are seeded locked and seeds.rb updates their content if needed, same approach they took in katello. Later when they are more stabilized I may move them here, but now I need to change them as I'm adding new features (client precreation instead of using validation.pem etc).